release-notes/3.0/preview/api-diff/preview5/standalone-packages/3.0-preview5-standalone-packages_System.Threading.Channels.md

```diff
@@ -0,0 +1,12 @@
+# System.Threading.Channels
```

Comment:
nit: Maybe the filename should be just `3.0-preview5-_System.Threading.Channels.md` The standalone-package in the name is redundant given the folder name.